.github/workflows/updates/Tor.sh

```diff
@@ -2,7 +2,11 @@
 
 
 webVer="$(wget -qO- https://sourceforge.net/projects/tor-browser-ports/files | grep -F 'Download Latest Version' --after 1 | tail -n -1 | tr '<>\-_' '\n' | sed -n 8p)"
-armhf_url="https://sourceforge.net/projects/tor-browser-ports/files/${webVer}/tor-browser-linux-armhf-${webVer}_en-US.tar.xz/download"
-arm64_url="https://sourceforge.net/projects/tor-browser-ports/files/${webVer}/tor-browser-linux-armhf-${webVer}_en-US.tar.xz/download"
+armhf_url="https://sourceforge.net/projects/tor-browser-ports/files/${webVer}/tor-browser-linux-armhf-${webVer}_ALL.tar.xz/download"
+arm64_url="https://sourceforge.net/projects/tor-browser-ports/files/${webVer}/tor-browser-linux-armhf-${webVer}_ALL.tar.xz/download"
+
+if [ -d ~/tor-browser/ ];then
+  mv ~/tor-browser ~/tor-browser_en-US
+fi
```

Comment:
This .github/workflows/updates/Tor.sh script is run on a virtual machine on github.com. It's primary job is to edit actual app scripts with new version numbers automatically. Please remove the lines here that move the tor-browser folder, as this folder will never exist on github's vm.